### DISCUSSION TITLE
Exploring the Potential of Memphis.dev for Real-Time Use Cases

### DISCUSSION THREAD

**Discussion by AndryHTC** [superstreamlabs/memphis#1106] [Q&A]
Hello Memphis Dev Team and community,

I have recently spent a good amount of time exploring different technologies for Change Data Capture (CDC) streaming, notably Debezium and Conduit.io. While these technologies offer powerful solutions, I've found their developer experience (DX) somewhat wanting.

Recently, I came across memphis.dev and have been genuinely impressed with the DX it offers. Although I understand that memphis is not typically compared to traditional CDC streaming solutions and is more in line with services such as Kafka, I am curious to explore its potential for my specific use cases. These include:

- Materialized Views
- Real-time Analytics
- Real-time Dashboards
- Real-time Queries
- Mobile App Synchronization

Given the developer-friendly approach of memphis, I am interested to understand how it could be leveraged in these areas.

In addition, I am also looking for alternatives to Debezium and Conduit.io that may offer a better DX, similar to what memphis delivers. I am open to suggestions for solutions that could provide comparable functionality in a more developer-friendly manner.

Looking forward to hearing from the team and the broader community on this. Your insights and experiences would be invaluable in helping me find the best path forward.

Best,
Andrea

**Comment by yanivbh1** [comment]
Hey @AndryHTC great to have you here, and thank you for finding Memphis helpful!
You mentioned multiple use cases, yes, Memphis can support most of them. If you are focusing on CDC pipelines, have you got the chance to try our latest series on the subject? https://memphis.dev/blog/part-1-integrating-debezium-server-and-memphis-dev-for-streaming-change-data-capture-cdc-events/

Besides CDC and in regards to the other use cases, if you can share more information on the greater goal, I will be happy to help more!

**Reply by AndryHTC** [reply]
Hi @yanivbh1,

Thanks for the quick reply and for the link to the series on integrating Debezium and memphis.dev for streaming Change Data Capture (CDC) events. I will certainly look into this.

However, my primary focus was to understand the possibilities of leveraging memphis.dev with CDC tools that put a stronger emphasis on user experience, similar to what Memphis provides. While Debezium is undoubtedly powerful, I've found it somewhat wanting when it comes to the developer experience (DX). 

Are there other tools that you could suggest which may align better with the DX offered by Memphis, while still providing robust CDC capabilities? The idea is to create a smoother, more intuitive development process for our team, while still achieving our use case goals for reactive programming and UI state management.

Thanks again for your assistance.

Best,
Andrea

**Comment by yanivbh1** [comment]
Adding @rnowling-memphis for the discussion as well, as he might have an experience with tools other than debezium.
